src/Plugin/StringMessageInitializerPlugin.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/*
+ * This file is part of the prooph/service-bus.
+ * (c) 2014-2015 prooph software GmbH <contact@prooph.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * Date: 10/20/15 - 11:40 AM
+ */
+
+namespace Prooph\ServiceBus\Plugin;
+
+use Prooph\Common\Event\ActionEvent;
+use Prooph\Common\Event\ActionEventEmitter;
+use Prooph\Common\Event\ActionEventListenerAggregate;
+use Prooph\Common\Event\DetachAggregateHandlers;
+use Prooph\ServiceBus\MessageBus;
+
+final class StringMessageInitializerPlugin implements ActionEventListenerAggregate
```

Comment:
i am not sure if we wan't to have a plugin for this. This behaviour should be part of the service bus by default. Thoughts @codeliner ?